<?php

echo <<< EOT


<div id='doc_acc'>
  <h3>Background</h3>
    <div class="doc">
      <p class='doc'>
        Currently, the publicly-available data sources accessed by BoB are datasets available in ArrayExpress, Gene Expression Omnibus (GEO), The Cancer Genome Atlas (TCGA)
        and Cancer Cell Line Encyclopedia (CCLE). BoB provides you with the means to conduct exploratory and in-depth analyses of transcriptomic,
        sequencing, genomic and mutation data obtained from both tissues and cell lines.
        <br><br>
        You have access to comprehensive analytical modalities from which you can conduct principal component analyses; estimate the tumour purity
        of samples; view the expression of gene(s) of interest across different biological conditions;
        conduct correlation analyses; visualise copy number aberrations and investigate mutational profiles. Furthermore, multidimensional data
        is integrated so that you can view data generated by different technologies from a single plot. This is crucial in understanding not only
        the different alterations but also the relationships between them.

        All figures are presented in an interactive and downloadable format. This means that all the figures are dynamic, allowing you to zoom in/out,
        focus on areas of interest, visualise the annotation of data points, and exclude or include samples in pre-defined biological groups.
        Furthermore, for each dataset the results from tumour purity estimates are also available to download
        n tab delimited format (target.txt).
      </p>
    </div>
    <h3>Define your analysis</h3>
      <div class="doc">
        <p class='doc'>
          An overview of the data types and analyses available in BoB is available from the homepage.
          You are presented with three tabs, representing the three data sources available currently.<br>
          <b>PubMed:</b> An automated data selection and retrieval system has been implemented. Publications of interest are identified from PubMed.
          Gene Expression Omnibus or ArrayExpress identifiers are used to establish computational links between the literature and any associated data.
          If data is available in the public domain, the system accesses the repository and downloads the relevant data files.
          These are fed into the relevant analytical pipelines and made available from the PubMed tab.<br>
          <b>The Cancer Genome Atlas (TCGA):</b> The TCGA consortium is dedicated to the systematic study of alterations in a variety of human cancers.
          It has made publicly available DNA copy number, mRNA expression, methylation and mutation data, alongside its associated clinical data
          for a range of cancer types/subtypes. Currently, mRNA expression and mutation data are available for analysis, with genomic data to follow shortly.<br>
          <b>Cancer Cell Line Encyclopedia (CCLE):</b> DNA copy number, mRNA expression and mutation data for pancreatic cancer cell lines are available from this tab.
          <br><br>
          Once you have selected a source, you will be directed to a page from which you will be able to conduct multiple analyses:
          principal component analysis (PCA), estimates of tumour purity, gene expression plots, correlation analyses, survival analyses and integrative analyses.
        </p>
      </div>
  <h3>Principal component analysis</h3>
    <div class="doc">
      <p class='doc'>
        Principal component analyses (PCA) allow you to visualise the global structure of the data and identify key “components” of variation.
        <br><br>
        For each dataset, scatterplots representing (A) the first two and (B) the first three principal components of the data are presented.
        In this instance, the dataset represents inflammatory pancreatic cancer (IBC) subtypes.
        Each data point represents the orientation of a single sample in the transcriptomic space projected on the PCA, with different colours
        indicating the biological group to which each sample belongs. The percentage values in brackets on each axis indicate the amount of variance
        in the data explained by the corresponding principal component. (C) The global variability of the data can also be assessed from the histogram.
        Here, you can identify the fraction of total variance (y-axis) attributed to each principal component (x-axis).
      </p>
      <br><br>
      <center> <img class="doc" src="../images/doc/pca.png"> </center>
    </div>
  <h3>Estimates of tumour purity</h3>
    <div class="doc">
      <p class='doc'>
        Cancer samples frequently contain a small proportion of infiltrating stromal and immune cells that may not only confound the
        tumour signal in molecular analyses but may also have a role in tumourigenesis and progression. We apply an algorithm that
        uses gene expression data to calculate stromal score, immune score and estimate score, and infer tumour purity from these values.
        <br><br>
        (<b>A</b>) A three-dimensional scatterplot displaying the stromal score, immune score and estimate score for each sample is provided.
        (<b>B</b>) The tumour purity estimates (percentages) for each sample are provided in a dynamic table that can be filtered by key words
        or tumour purity values.

        <center> <img class="doc" src="../images/doc/estimates.png"> </center>
      </p>
    </div>
  <h3>Gene expression plots</h3>
    <div class="doc">
      <p class='doc'>
        From this tab, you are able to view how the expression of your gene(s) varies across different biological conditions.
        To provide a comprehensive overview of expression values across the biological groups, results are presented as both
        summarised and a sample views (box-and whisker plot and barplot, respectively).
        <br><br>
        Here we show examples of results obtained using the CCLE dataset. (<b>A</b>) From the box-and-whisker plot, can view if, and how,
        the expression of your gene of interest varies across the different biological groups. (<b>B</b>) To facilitate the understanding
        of any differences, a barplot in which the expression values of individual samples (bars) is also provided.
        The samples are clustered by biological group and ordered within each group by expression value.

        <center> <img class="doc" src="../images/doc/gene_expression.png"> </center>
      </p>
    </div>
  <h3>Correlation analysis</h3>
    <div class="doc">
      <p class='doc'>
        Pairwise correlations of your genes of interest are conducted.
        <br><br>
        For each comparison, the Pearson Product Moment Correlation Coefficient (PMCC) and p-value is calculated and presented in a heatmap (<b>A</b>).
        The colour of each cell represents the correlation coefficient for each comparison.
        The correlation value calculated for each pairwise comparison can be viewed by hovering over the cells.

        <center> <img class="doc" src="../images/doc/correlations.png"> </center>
      </p>
    </div>
  <h3>Survival analysis</h3>
    <div class="doc">
      <p class='doc'>
        From the survival tab, you are able to visualise the relationship between survival and 
        gene(s) of interest. For the latter, a univariate model is applied to the survival data and samples are assigned to risk
        groups based on the median expression value for the gene of interest.
        <br>
        <br>
        Kaplan Meier plots are generated to view the relationship
        between survival and a gene(s) of interest.

        <center> <img class="doc" src="../images/doc/survival.png"> </center>
      </p>
    </div>
    <h3>Identification of frequently mutated genes</h3>
      <div class="doc">
        <p class='doc'>
          Advances in sequencing technologies allow for the systematic characterisation of mutations in pancreatic cancer.
          A summary of the top mutations identified in pancreatic cancer can be accessed from the Mutation analysis tab.
          <br><br>
          MuTect2 is applied to call somatic single nucleotide variants from matched tumour-normal samples and isolate the top mutated genes.
          <b>(A)</b> The top mutation events are presented as a heatmap, with barplots indicating the number of <b>(B)</b> mutations per sample
          and <b>(C)</b> type of mutations per gene also generated. <b>(D)</b> The clinical covariates associated with each sample can
          be visualised below the heatmap.

          <center> <img class="doc" src="../images/doc/oncoprint.png"> </center>
        </p>
      </div>
  <h3>Integration of multidimensional data</h3>
    <div class="doc">
      <p class='doc'>
        For instances in which multiple technologies have been applied to the same sample/patient, you have the opportunity to integrate the different types of data,
        allowing for the visual exploration of the relationship between them.
        <br>
        <br>
        Once a gene of interest has been specified, you will be presented with up to four plots. The number of results generated is dependent on the data that is available.
        If information about gene expression and copy number alterations is available, (<b>A</b>) a scatterplot of expression values (y-axis) against copy
        number values (x-axis) and (<b>B</b>) a box-and-whisker plot displaying the expression values (y-axis) of samples within categorised copy number
        alterations (x-axis) are generated. For these plots, the biological group of each sample is indicated by the colour of the data point.
        If mutational information is also available, an additional (<b>C</b>) scatterplot and (<b>D</b>) box-and-whisker plot is generated, where the mutational
        status of each sample is indicated by the colour of the data point.

        <center> <img class="doc" src="../images/doc/mut.png"> </center>
      </p>
    </div>
</div>

<script>LoadDocAcc();</script>

EOT;

?>
